cognite_toolkit/modules/cdf_data_pipeline_asset_default/extraction_pipeline/source_extraction_pipeline.yaml

```diff
@@ -0,0 +1,14 @@
+external_id: 'ep-src:asset:{{location_name}}:{{source_name}}'
+name: 'src:asset:{{location_name}}:{{source_name}}'
+data_set_external_id: 'asset:{{location_name}}'
+description: 'Asset source extraction pipeline with configuration for DB extractor reading data from {{location_name}}:{{source_name}}'
+rawTables:  [
+    dbName: 'src-asset-{{location_name}}:{{source_name}}',
+    tableName: 'assets'
+  ]
+
+source: '{{source_name}}'
+documentation: "The DB Extractor is a general database extractor that connects to a database, executes one or several queries and sends the result to CDF RAW.\n\nThe extractor connects to a database over ODBC, which means that you need an ODBC driver for your database. If you are running the Docker version of the extractor, ODBC drivers for MySQL, MS SQL, PostgreSql and Oracle DB are preinstalled in the image. See the example config for details on connection strings for these. If you are running the Windows exe version of the extractor, you must provide an ODBC driver yourself. These are typically provided by the database vendor.\n\nFurther documentation is available [here](./docs/documentation.md)\n\nFor information on development, consider the following guides:\n\n * [Development guide](guides/development.md)\n * [Release guide](guides/release.md)"
+
+config: "logger:\n  console:\n    level: INFO\n\n  file:\n    level: INFO\n    path: \"file.log\"\n\n\n# List of databases\ndatabases:\n  - type: odbc\n    name: postgres\n    connection-string: \"DSN={MyPostgresDsn}\"\n\n\n# List of queries\nqueries:\n  - name: test-postgres\n    database: postgres\n\n    query: >\n      SELECT\n        *\n      FROM\n        mytable\n      WHERE\n        {incremental_field} >= '{start_at}'\n      ORDER BY\n        {incremental_field} ASC\n\n    incremental-field: \"id\"\n    initial-start: 0\n\n    destination:\n      type: raw\n      database: \"db-extractor\"\n      table: \"postgres\"\n\n    primary-key: \"{id}\""
```

Comment:
Is it possible to format this for easier reading without breaking the yaml parsing? I.e. config:| (and then remove \n)
Ref https://yaml-multiline.info